dds_web/templates/mail/deletion_request.html

```diff
@@ -15,15 +15,18 @@
       </p>
       <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 10px;">
         We appreciate you using our services and hope that the received data as well as the collaboration as a whole has
-        met your expectations. 
-
-        {% if projectnames is not none %}
-        Currently, you are affiliated with those projects:
-        {{projectnames}}
-        {% endif %}
-        
+        met your expectations.
+      </p>
+      {% if projects is not none %}
+        <p style="font-family: arial, sans-serif; font-size: 14px; font-weight: normal; margin: 0; Margin-bottom: 10px;">
+        Currently, you are affiliated with these projects:
```

Comment:
I think we may want to add a list of these instead?
Just to make it easier to read?

Reply:
Done!